### ISSUE TITLE
`prevent-abbreviations` is too eager

### ISSUE THREAD

**Issue by dasZGFz** [sindresorhus/eslint-plugin-unicorn#367]
This code:
```js
const image = new Image();
image.src = '';
```
outputs this as an error:
```
The property src should be named source. A more descriptive name will do too.  unicorn/prevent-abbreviations
```

My current workaround is to disable eslint for that line but I wonder if there's a way to fix it in the plugin.

**Comment by garyking** [comment]
sindresorhus's own package, [execa](https://github.com/sindresorhus/execa), uses `env` as an option, yet this rule reports an error for that abbreviation, too. So I agree that sometimes the rule is too aggressive. It shouldn't report on arguments that we pass to a function, or values returned from a function. Since if those were functions that we created ourselves and were in our own scripts, then those occurrences would already be reported.
